src/test/java/it/cnr/isti/hpc/wikipedia/article/en/ArticleTest.java

```diff
@@ -185,7 +185,7 @@ public void testAnnotationsInLists() throws IOException {
 			}
 		}
 
-		assertThat(uris, hasItems("Yū_Hayami", "Takumi_Hayami", "Hayami_District,_Ōita", "Mokomichi_Hayami"));
+		assertThat(uris, hasItems("Yū_Hayami", "Takumi_Hayami", "Hayami_District,_Ōita", "Mokomichi_Hayami", "H2O:_Footprints_in_the_Sand"));
```

Comment:
How come there is a new item? Did you change the file `./src/test/resources/en/Hayami`?